### PR TITLE
Remove search result when input empty

### PR DESCRIPTION
**Pull request by nadine-piveteau**
Related to issue #4095

### CODE REVIEW

**Comment by gjn** [comment]
Please always provide a testlink so people can test it.

Code looks good to me.

**Comment by nadine-piveteau** [comment]
[test link](https://mf-chsdi3.int.bgdi.ch/shorten/78007ba47a)

try for instance to enter "Bahnhofstrasse" into the search bar and then remove fastly all letters with the backspace key. No search results should be displayed anymore.

File: src/components/search/partials/search.html [hunk 1]
```diff
@@ -11,10 +11,10 @@
     <i class="fa fa-search"></i>
     <button class="fa fa-remove-sign ga-icon ga-btn" ng-click="clearInput()"
        ng-mousedown="preClear($event)" ng-hide="query==''"></button>
-
     <span class="ga-search-dropdown"
           ng-show="restat.sets()"
-          ng-class="'ga-search-' + restat.sets()">
+          ng-class="'ga-search-' + restat.sets()"
+          ng-if="!(query==''||query=='undefined')">
```
Comment (oterral) [inline src/components/search/partials/search.html]: There is already a `ng-show`, you should use it: 
```
ng-show="query && restat.sets()"
```

Comment (nadine-piveteau) [reply src/components/search/partials/search.html]: Ok thx a lot! I accordingly adapted the unittests. Test link is updated.

**Pull request merged**